### DISCUSSION TITLE
Do we have any API in Jgit equivalent to SVN's SvnSetProperty

### DISCUSSION THREAD

**Discussion by pratap043** [git-lfs/git-lfs#5036] [Q&A]
Hi Team,

We are trying to implement the Property setting and reading for a file in GitHub using Jgit API, similar to what we have in SVN (SvnSetProperty API). I am not able to find the API/solution for the same. 
Any help is appreciated in this regards.

**Comment by bk2204** [comment]
Hey,

This discussion is for the open source Git LFS project and doesn't have anything to do with Git itself or JGit.  You'd want to find the appropriate forum for JGit and ask this question there instead.
